### ISSUE TITLE
Bug: documentation refers to convertToPlainTextUtils which does not exist on codebase

### ISSUE THREAD

**Issue by lander854** [facebook/lexical#1926]
Lexical version:
latest
## Steps To Reproduce

1. look for convertToPlainTextUtils
2. it does not exist

how else can  we convert to something the backend can understand?

**Comment by lander854** [comment]
https://lexical.dev/docs/api/lexical-markdown

**Comment by zurfyx** [comment]
Good catch! Feel free to put a PR to edit the Wiki. Currently, there's only one way of the conversion, see `convertFromPlainTextUtils`. @fantactuka and @kraisler are actively working on the bidirectional conversion
